rustls/examples/internal/bogo_shim.rs

```diff
@@ -177,6 +178,19 @@ fn load_key(filename: &str) -> PrivateKeyDer<'static> {
     keys.pop().unwrap().into()
 }
 
+fn load_root_certs() -> Arc<RootCertStore> {
+    let mut roots = RootCertStore::empty();
+
+    // this is not actually used by the tests, but must be non-empty
+    for c in load_cert("cert.pem") {
+        roots
+            .add(c)
+            .expect("cannot convert cert.pem to trust anchor");
+    }
```

Comment:
given the comment maybe this could just be:
```suggestion
    // this is not actually used by the tests, but must be non-empty
   roots.add_parsable_certificates(load_cert("cert.pem"));
```